Yoast/wordpress-seo
# Use sources for social images

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:
* Set the sitewide default image for Open Graph.

### Open Graph image
* Tests for a post
   * Set a image for 'Facebook' in the settings and confirm is it visible as og tag.
   * Remove the image and set a featured image and verify it is in the og tag.
   * Remove the featured image and set an image in the content and do the verification again.
   * Remove the image(s) in the content and verify the default image is used in the tag.
* Tests for an attachment
  * Have an attachment as image and verify it is rendered as OG tag. Make sure you don't redirect the attachments (is in your search appearance - media settings)
  * Because of an existing bug you can't set images at the moment - this should be fixed in a separate pull
* Test for a term
   * Set a image for 'Facebook' in the settings and confirm is it visible as og tag.
   * Remove the image and verify the default image is used in the tag.
* Test for the home page
   * In the social menu set the image for the front page and verify it is present in the tag.
   * Remove the image and verify the default image is used in the tag.

### Twitter image
* For testing a the twitter images you can follow the OG scenarios but instead of setting the facebook image you have to set the twitter image. 
* When OpenGraph is enabled it will always fallback to the OG-logic.

### Twitter image with OpenGraph disabled
* Disable opengraph under menu social - opengraph
* Follow the same steps with the twitter image field as basis. It should never show the OG default image when no images are found.

## UI changes

## Documentation

## Quality assurance


Fixes #13593
